src/status_im/ui2/screens/chat/group_details/view.cljs

```diff
@@ -5,60 +5,97 @@
+            [reagent.core :as reagent]
+            [status-im2.common.home.actions.view :as actions]
+            [status-im.ui2.screens.common.contact-list.view :as contact-list]
+            [oops.core :refer [oget]]))
 
 (defn back-button []
   [quo2/button {:type                :grey
                 :size                32
+                :width               32
                 :style               {:margin-left 20}
                 :accessibility-label :back-button
-                :on-press            #(rf/dispatch [:navigate-back])
-                :icon                true}
-   :i/arrow-left])
+                :on-press            #(rf/dispatch [:navigate-back])}
+   [quo2/icon :i/arrow-left {:color (colors/theme-colors colors/neutral-100 colors/white)}]])
 
 (defn options-button []
-  [quo2/button {:type                :grey
-                :size                32
-                :style               {:margin-right 20}
-                :accessibility-label :options-button
-                :icon                true}
-   :i/options])
+  (let [group (rf/sub [:chats/current-chat])]
+    [quo2/button {:type                :grey
+                  :size                32
+                  :width               32
+                  :style               {:margin-right 20}
+                  :accessibility-label :options-button
+                  :on-press            #(rf/dispatch [:bottom-sheet/show-sheet
+                                                      {:content (fn [] [actions/group-details-actions group])}])}
+     [quo2/icon :i/options {:color (colors/theme-colors colors/neutral-100 colors/white)}]]))
+
+(defn top-buttons []
+  [rn/view {:style {:flex-direction     :row
+                    :padding-horizontal 20
+                    :justify-content    :space-between}}
+   [back-button] [options-button]])
 
 (defn count-container [count]
   [rn/view {:style (style/count-container)}
    [quo2/text {:size   :label
                :weight :medium
                :style  {:text-align :center}} count]])
 
-(defn prepare-members [members]
-  (let [admins (filter :admin? members)
-        online (filter #(and (not (:admin? %)) (:online? %)) members)
-        offline (filter #(and (not (:admin? %)) (not (:online? %))) members)]
-    (vals (cond-> {}
-            (seq admins) (assoc :owner {:title "Owner" :data admins})
-            (seq online) (assoc :online {:title "Online" :data online})
-            (seq offline) (assoc :offline {:title "Offline" :data offline})))))
-
 (defn contacts-section-header [{:keys [title]}]
   [rn/view {:style {:padding-horizontal 20 :border-top-width 1 :border-top-color colors/neutral-20 :padding-vertical 8 :margin-top 8}}
    [quo2/text {:size   :paragraph-2
                :weight :medium
                :style  {:color (colors/theme-colors colors/neutral-50 colors/neutral-40)}} title]])
 
+(def added (reagent/atom ()))
+
+(defn contact-requests-sheet [group]
+  [:f>
```

Comment:
better to move atom here 

```
(defn contact-requests-sheet [group]
  (let [added (reagent/atom ())]
    (fn []
       [:f>
          (fn []
              (let [{window-height :height}
```

Reply:
Because there is a list, and the bottom container that includes the button needs to always be floating on top of the list and the bottom.